builders/build/substrate-api/polkadot-js-api.md

```diff
@@ -325,12 +316,99 @@ You can view the [complete script on GitHub](https://raw.githubusercontent.com/P
 !!! note
     You can check out all of the available functions for the `parachainStaking` module by adding `console.log(api.tx.parachainStaking);` to your code.
 
-## Custom RPC Requests {: #custom-rpc-requests }
+## Substrate and Custom JSON-RPC Endpoints {: #substrate-and-custom-json-rpc-endpoints }
 
 RPCs are exposed as a method on a specific module. This means that once available, you can call any rpc via `api.rpc.<module>.<method>(...params[])`. This also works for accessing Ethereum RPCs using Polkadot.js API, in the form of `polkadotApi.rpc.eth.*`.
 
-You can check for a list of exposed RPC endpoints by calling `api.rpc.rpc.methods()`, which is the list of known RPCs the node exposes. 
+Some of the methods availabe through the Polkadot.js API interface are also available as JSON-RPC endpoints on Moonbeam nodes. This section will provide some examples; you can check for a list of exposed RPC endpoints by calling `api.rpc.rpc.methods()` or the `rpc_methods` endpoint listed below. 
+
+- [`methods()`](https://polkadot.js.org/docs/substrate/rpc/#methods-rpcmethods){target=_blank}: `RpcMethods`
```

Comment:
I'm a little confused about the `RpcMethods` part, what are we trying to say with this? It doesn't seem like it's part of the code so just wasn't sure what it is for

Same with `SignedBlock` and `BlockHash` below

Reply:
Those are what the methods return. I've merged that with the summary part.